scripts/docker-commands.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# create subshell to wait as the local contract network spins up, then deploy the token to the network
+(sleep 60 && cd saddle-token && npx hardhat deploy --network localhost;) & \
+npm run start;
```

Comment:
Is it starting up this AND the saddle-contract start script?

Reply:
because of the sleep, the contract repo starts (creates a network and deploys its contracts), and then the token repo deploys its contracts to the existing network.